mlflow/server/__init__.py

```diff
@@ -17,7 +17,6 @@
 # NB: These are internal environment variables used for communication between
 # the cli and the forked gunicorn processes.
 BACKEND_STORE_URI_ENV_VAR = "_MLFLOW_SERVER_FILE_STORE"
-REGISTRY_STORE_URI_ENV_VAR = "_MLFLOW_SERVER_REGISTRY_STORE"
```

Comment:
WHat is the purpose of this change?

Reply:
looks like this env is designed to be only used for mlflow server. Will revert the change.